WreckEngine/TextEntity.cpp

```diff
@@ -1,6 +1,17 @@
 #include "TextEntity.h"
 
 TextEntity::TextEntity(const std::string _message, const std::string& _font, Text::Justify _vertical, Text::Justify _horizontal, const uint32_t height, const uint32_t width)
-	: Entity(), font(Text::loadWinFont(_font, height, width)), message(_message), vertical(_vertical), horizontal(_horizontal) {}
+    : Entity()
+    , vertical(_vertical)
+    , horizontal(_horizontal)
+    , inst(make_shared<Text::Instance>())
```

Comment:
Hmm... Well, is there ever going to be a situation where you have something like `TextEntity text1 = ...; TextEntity text2 = text1;` and will assume that `text2` would be identical to `text1` but be able to have its own text, position, offset, etc?